### PR TITLE
Added resolved issues list for 4.5.4

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_4.5.md [hunk 1]
```diff
@@ -3,6 +3,32 @@ title: 128T 4.5 Release Notes
...
+------
+- **I95-37774 Detached Flow Collision:** A rare case that allowed a flow collision with a detached flow. 
+------
+-**I95-38084 SNMP failure alarm after upgrade:** Resolved an issue where the snmpd service was starting before the configuration had been received causing the failure alarm.  
+------
+- **I95-38152 Access Policy field in GUI with wrong title:** Corrected the title in the Access Policy for Service field.
+------
+- **I95-38175 Deleting an invalid interface from the router logs an error:** Resolved an issue where a non-forwarding interface would fail to be removed from Linux when deleted from configuration.
+------
+-**I95-38194 Conductor cannot see peer path stats in GUI:** Teh web server now makes an explicit version check to verify the correct data is sent.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.5.md:22]: Typo.  "The"

File: docs/release_notes_128t_4.5.md [hunk 2]
```diff
@@ -3,6 +3,32 @@ title: 128T 4.5 Release Notes
...
+------
+-**I95-38084 SNMP failure alarm after upgrade:** Resolved an issue where the snmpd service was starting before the configuration had been received causing the failure alarm.  
+------
+- **I95-38152 Access Policy field in GUI with wrong title:** Corrected the title in the Access Policy for Service field.
+------
+- **I95-38175 Deleting an invalid interface from the router logs an error:** Resolved an issue where a non-forwarding interface would fail to be removed from Linux when deleted from configuration.
+------
+-**I95-38194 Conductor cannot see peer path stats in GUI:** Teh web server now makes an explicit version check to verify the correct data is sent. 
+------
+- **I95-38196 One way Audio issue:** Resolved a session collision issue resulting in a lost service path.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.5.md:24]: This should not be labeled one way audio. That's an artifact of the particular use case.
Suggested rewording:
I95-38194 Session Collision: Resolved a session collision where a flow is setup at the same time from both directions results in only one of the two flows being established.

File: docs/release_notes_128t_4.5.md [hunk 3]
```diff
@@ -3,6 +3,32 @@ title: 128T 4.5 Release Notes
...
+------
+- **I95-39223 Using a question mark on a poorly written command dumps a stack trace:** Added checks against syntax in Command line commands.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.5.md:26]: Issue ID is incorrect. It should be I95-38223
Suggested rewording:
Using a question mark on a command that does not exist creates a stack trace: ...

File: docs/release_notes_128t_4.5.md [hunk 4]
```diff
@@ -3,6 +3,32 @@ title: 128T 4.5 Release Notes
...
+------
+- **I95-38175 Deleting an invalid interface from the router logs an error:** Resolved an issue where a non-forwarding interface would fail to be removed from Linux when deleted from configuration.
+------
+-**I95-38194 Conductor cannot see peer path stats in GUI:** Teh web server now makes an explicit version check to verify the correct data is sent. 
+------
+- **I95-38196 One way Audio issue:** Resolved a session collision issue resulting in a lost service path.
+------
+- **I95-39223 Using a question mark on a poorly written command dumps a stack trace:** Added checks against syntax in Command line commands. 
+------
+- **I95-38230 Support for off-subnet-reverse-arp-mac-learning:** A new flag has been added to the network interface to turn on this feature.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.5.md:28]: This needs updates and a reference in the cli_elements

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_4.5.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_4.5.md:22]: resolved

File: docs/release_notes_128t_4.5.md (hunk 2 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_4.5.md:24]: Revised, using an edited version of your comment.

File: docs/release_notes_128t_4.5.md (hunk 3 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_4.5.md:26]: revised

File: docs/release_notes_128t_4.5.md (hunk 4 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_4.5.md:28]: revised, but waiting on confirmation/info from Abilash before pushing doc back up.

**Review by MichaelBaj — DISMISSED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by jeffmart — APPROVED** [review]

**Pull request merged**